content/learn/book/contributing/docs/_index.md

```diff
@@ -4,6 +4,7 @@ weight = 2
 sort_by = "weight"
 template = "book-section.html"
 page_template = "book-section.html"
+insert_anchor_links = "left"
```

Comment:
left anchors make the heading alignment feel off. 

I'd prefer something closer to this:
![image](https://user-images.githubusercontent.com/2694663/92046433-61eb0d80-ed37-11ea-8d64-067c7fc6b45d.png)

```css
.bevy-link {
    font-size: 1.3rem;
    vertical-align: top;
    margin-left: 0.2rem;
}
```

```html
<a class="bevy-link" href="#{{ id }}" aria-label="Anchor link for: {{ id }}">🔗</a>
```

```
insert_anchor_links = "right"
```

Reply:
I started off on the right :) I didn't like the way it looked, so I pushed it to the left for a little more prominence. Putting it back on the right is fine.

I like this style of link icon, but there's no implementation in Unicode, so if you want SVG it's the perfect opportunity for me. Which shape would you prefer? I could do both and see how it pans out at font size.

![internal_link](https://user-images.githubusercontent.com/12194690/92183193-3c1d5000-ee80-11ea-8164-9ac02d4d59eb.png)
<img src="https://user-images.githubusercontent.com/12194690/92183262-70910c00-ee80-11ea-912a-1897c4bf5c1f.jpg" width="200" height="200">

Reply:
I wish headings could just automatically be links, instead of requiring a link icon. If we can make that work, that would be my preference.

The link icons above are definitely a common design, but they're a bit noisy for my taste. I like the simplicity of the "infinity" style logo. Or maybe even just a `#` character ... thats minimal and matches the anchor link style. The `#` wouldn't even require an SVG.

In general I want to reduce "noise" as much as possible and use a minimal "text first" / "content first" approach.

Reply:
I replaced with a custom link svg, that looks like the unicode one. This means it can now take the $default_color. The downside is that the svg is not dynamically scalable within Zola. However, I used three sizes that correspond to h2 a svg {}, h3 a svg {}, and h4 a svg {} repectively. I hide the sizes that shouldn't display in each case (`templates/anchor-link.html`) Short story, the symbol varies in size with respect to each heading type.